features/trigger.feature

```diff
@@ -0,0 +1,54 @@
+@ignore
+Feature: Trigger
```

Comment:
There's actually at least three different features described in this feature file, I'm not sure if you want them all in one or want to break them up: rebuild on/reverse trigger, fan-out, and standard intra-workflow triggers. Will fan-in be added to this, or a different test? The scenarios in this file are mostly dealing with the rebuild on/reverse trigger feature.

Reply:
Fan-in and Fan-out scenarios are different and is already in capture for `workflow`. Fan-out scenario is reemphasized here to capture the fact that there should be only one event. All the scenarios in this feature file arise from the successful completion of a single external pipeline's job.